README.md

```diff
@@ -0,0 +1,10 @@
+- 👋 Hi, I’m @bodssfh
+- 👀 I’m interested in ...
+- 🌱 I’m currently learning ...
+- 💞️ I’m looking to collaborate on ...
+- 📫 How to reach me ...
+
+<!---
+bodssfh/bodssfh is a ✨ special ✨ repository because its `README.md` (this file) appears on your GitHub profile.
+You can click the Preview link to take a look at your changes.
+--->
```

Comment:
Steps to Implement

	1.	Create OAuth Application: Use the provided information to register an OAuth application.
	2.	Implement the Scraping Logic: Continuously scrape car and motorcycle listings from willhaben.at.
	3.	Create Flask Application: Serve the scraped data through a Flask web application.
	4.	Setup Project for Deployment: Ensure the project is ready for deployment on Heroku.